api/v1/checks.go

```diff
@@ -660,8 +663,10 @@ type SmbCheck struct {
 	//MaxAge the latest object should be younger than defined age
 	MaxAge string `yaml:"maxAge,omitempty" json:"maxAge,omitempty"`
 	//MinCount the minimum number of files inside the searchPath
-	MinCount        int    `yaml:"minCount,omitempty" json:"minCount,omitempty"`
-	Description     string `yaml:"description,omitempty" json:"description,omitempty"`
+	MinCount    int    `yaml:"minCount,omitempty" json:"minCount,omitempty"`
+	Description string `yaml:"description,omitempty" json:"description,omitempty"`
+	//DisplayTemplate represents the output format for the results. Example 'File Age: [[.age]]'
```

Comment:
Please make spacing here consistent in the doc?

Reply:
In the comments?